### PR TITLE
Implement svc 0x04, 0x06, 0x53, fix debug-related definitions in svc.h

### PR DESCRIPTION
**Pull request by TuxSH**

### CODE REVIEW

**Comment by fincs** [comment]
LGTM

File: libctru/include/3ds/svc.h [hunk 1]
```diff
@@ -140,10 +140,10 @@ typedef struct {
 
 /// Reasons for an exit thread event.
 typedef enum {
-	EXITTHREAD_EVENT_NONE              = 0, ///< No reason.
+	EXITTHREAD_EVENT_EXIT              = 0, ///< Thread exited.
 	EXITTHREAD_EVENT_TERMINATE         = 1, ///< Thread terminated.
-	EXITTHREAD_EVENT_UNHANDLED_EXC     = 2, ///< Unhandled exception occurred.
-	EXITTHREAD_EVENT_TERMINATE_PROCESS = 3, ///< Process terminated.
+	EXITTHREAD_EVENT_EXIT_PROCESS      = 2, ///< Process exited either normally or due to an uncaught exception.
```
Comment (mtheall) [inline libctru/include/3ds/svc.h]: "Thread" instead of "process'? Same question for line below.

File: libctru/include/3ds/svc.h [hunk 2]
```diff
@@ -614,9 +630,16 @@ Result svcCreateProcess(Handle* out, Handle codeset, const u32 *arm11kernelcaps,
...
+ * @param[out] processorid Pointer to store the ID of the process's ideal processor.
+ * @param process Handle of the process.
+ */
+Result svcGetProcessIdealProcessor(Handle process, s32 processorid);
+
+/**
  * Sets a process's ideal processor.
  * @param process Handle of the process.
- * @param processorid ID of the thread's ideal processor.
+ * @param processorid ID of the process's ideal processor.
```
Comment (mtheall) [inline libctru/include/3ds/svc.h]: Is this affinity for the main thread of a process?

File: libctru/include/3ds/svc.h (hunk 1 above)
Comment (TuxSH) [inline libctru/include/3ds/svc.h]: No, this is when a process ends and its threads are terminated.

File: libctru/include/3ds/svc.h (hunk 2 above)
Comment (TuxSH) [inline libctru/include/3ds/svc.h]: No, it's taken from the KProcess structure.

**Pull request merged**